scripts/qa_crowbarsetup.sh

```diff
@@ -4695,7 +4695,7 @@ function onadmin_prepare_crowbar_upgrade
         safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
     else
         safely crowbarctl upgrade prepare
-        wait_for 300 5 "grep current_step $upgrade_progress_file | grep -v prepare" "prepare step to finish"
+        wait_for 300 10 "crowbarctl upgrade status | grep 'steps.prepare.status' | grep -q passed" "prepare step to finish"
```

Comment:
would this work as well?
`... | grep -q 'steps.prepare.status.*passed'
If so, I would recommend it, as it is shorter.